openshift/e2e-tests-openshift.sh

```diff
@@ -254,6 +283,50 @@ function run_e2e_tests(){
   return $failed
 }
 
+function run_rolling_upgrade_tests() {
+    header "Running rolling upgrade tests"
+    
+    local TIMEOUT_TESTS="20m"
+    failed=0
+
+    report_go_test -tags=preupgrade -timeout=${TIMEOUT_TESTS} ./test/upgrade \
+    --dockerrepo "${INTERNAL_REGISTRY}/${SERVING_NAMESPACE}" \
+    --kubeconfig $KUBECONFIG || failed=1
+
+    echo "Starting prober test"
+
+    # Make prober send requests more often compared with upstream where it is 1/second
+    sed -e 's/\(.*requestInterval =\).*/\1 200 * time.Millisecond/' -i vendor/knative.dev/pkg/test/spoof/spoof.go
```

Comment:
Why do we do that? Should we propose a change upstream to make this more configurable?

Reply:
I want to be sending requests more often than once per second to make sure the upgrade can be done even if we have a lot of requests and they don't fail. I'm not sure why upstream has this.

Reply:
Let's drive a change upstream then as well.